cassandra-plugins/src/main/java/co/cask/hydrator/plugin/batch/source/BatchCassandraSource.java

```diff
@@ -118,6 +119,15 @@ public void transform(KeyValue<Long, Row> input, Emitter<StructuredRecord> emitt
     emitter.emit(builder.build());
   }
 
+  @Override
+  public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
+    super.configurePipeline(pipelineConfigurer);
+
+    Schema inputSchema = pipelineConfigurer.getStageConfigurer().getInputSchema();
+    pipelineConfigurer.getStageConfigurer().setOutputSchema(inputSchema);
```

Comment:
The input schema for a source will always be null.  Should be setting the schema based on what is set in the config.  Something like:
```
try {
  Schema schema = Schema.parseJson(config.schema);
  pipelineConfigurer.getStageConfigurer().setOutputSchema(schema);
} catch (Exception e) {
  throw new IllegalArgumentException("Invalid schema: " + e.getMessage(), e);
}
```